int16 screenId
int16 feedId
---
int16 success
